handler/population-types_test.go

```diff
@@ -24,7 +24,7 @@ func TestEndpointRoot(t *testing.T) {
 		cantabularClient := fakeCantabularClient{
 			listDatasetsReturnData: []string{"Thing one", "Thing two"},
 		}
-		handler := handler.NewPopulationTypes(responder, &cantabularClient)
+		handler := handler.NewPopulationTypes(responder, &cantabularClient, nil, nil)
```

Comment:
Should we be testing the behavior of the other clients?

Reply:
I have added a test to show a failing dataset client, but I feel as if they
are covered by unit tests in their respective unit tests in api-clients-go.